### ISSUE TITLE
if "x$HAVE_DBUS"

### ISSUE THREAD

**Issue by WayneSallee** [lathiat/avahi#72]
Looks like a bug.

Squashes like a bug.
:-)

File:
`configure`

Looks like:

```
BUILD_CLIENT="no   (You need avahi-daemon and D-Bus!)"

if "x$HAVE_DBUS" = "xyes" ; then
    BUILD_CLIENT=yes
fi
```

Should have "test" added  be changed to:

```
BUILD_CLIENT="no   (You need avahi-daemon and D-Bus!)"

if test "x$HAVE_DBUS" = "xyes" ; then
    BUILD_CLIENT=yes
fi
```

After doing so I was able to run conifig and install without problems.

Wayne Sallee
Wayne@WayneSallee.com
http://www.WayneSallee.com

**Comment by rworkman** [comment]
Good catch; I just stumbled across this one too.

**Comment by lathiat** [comment]
Already fixed in #51 but is not in the 0.6.32 release, will be in the 0.6.33 release soon.  Thanks!
